dds_web/api/project.py

```diff
@@ -488,15 +488,23 @@ def format_project_dict(self, current_user):
         # Apply the filters
         user_projects = models.Project.query.filter(sqlalchemy.and_(*all_filters)).all()
 
+        researcher = False
+        if auth.current_user().role not in ["Super Admin", "Unit Admin", "Unit Personnel"]:
+            researcher = True
+
         # Get info for all projects
         for p in user_projects:
+            project_creator = p.creator.name
```

Comment:
I tried deleting the unituser and realised that this results in errors because `p.creator` is none and therefore `p.creator.name` cannot be collected. 
```suggestion
            project_creator = p.creator.name if p.creator else None
```